BlueMap/Chat/src/main/resources/config.yml

```diff
@@ -6,3 +6,4 @@ read-only: false
 
 max-message-count: 100
 web-chat-prefix: "[web]"
+empty-chat-message: "No messages yet"
```

Comment:
Does it make sense for this one message to be configurable when rest of the UI isn't? Maybe leave this out of this PR and look into translation/customization options later?

Reply:
Yeh that is probably better